src/utils.rs

```diff
@@ -208,11 +208,14 @@ pub mod test_utils {
 
     macro_rules! cairo_runner {
         ($program:expr) => {
-            CairoRunner::new(&$program, "all").unwrap()
+            CairoRunner::new(&$program, "all", false).unwrap()
         };
         ($program:expr, $layout:expr) => {
             CairoRunner::new(&program, $layout).unwrap()
```

Comment:
I think it's missing a parameter here:
```suggestion
            CairoRunner::new(&program, $layout, false).unwrap()
```